### PR TITLE
Add polish translations

### PR DESCRIPTION
**Pull request by marcelib**

### CODE REVIEW

File: datepicker/languages.js [hunk 1]
```diff
@@ -97,5 +97,14 @@ export default {
       'date': 'Izberite datum',
       'dateRange': 'Izberite razpon med 2 datumoma'
     }
+  },
+  'pl': {
+    'days': ['Niedz', 'Pon', 'Wt', 'Śr', 'Czw', 'Pt', 'Sob'],
+    'months': ['Sty', 'Lut', 'Mar', 'Kwi', 'Maj', 'Cze', 'Lip', 'Sie', 'Wrz', 'Paź', 'Lis', 'Gru'],
+    'pickers': ['następne 7 dni', 'następne 30 dni', 'wcześniejsze 7 dni', 'wcześniejsze 30 dni'],
```
Comment (MJKruszewski) [inline datepicker/languages.js]: Maybe 'poprzednie 7 dni' etc. would be more appropriate ?

Comment (cichy380) [reply datepicker/languages.js]: I suggest shorten day of the week names and use of two-letter calendar symbols

```
  'pl': {
    'days': ['Nd', 'Pn', 'Wt', 'Śr', 'Cz', 'Pt', 'So'],
    'months': ['Sty', 'Lut', 'Mar', 'Kwi', 'Maj', 'Cze', 'Lip', 'Sie', 'Wrz', 'Paź', 'Lis', 'Gru'],
    'pickers': ['następne 7 dni', 'następne 30 dni', 'poprzednie 7 dni', 'poprzednie 30 dni'],
```

**Comment by mengxiong10** [comment]
https://github.com/mengxiong10/vue2-datepicker/pull/62#issuecomment-442685399

**Comment by mengxiong10** [comment]
Thanks for PR.
v3.x added the language.

**Pull request closed without merging**